metaci/release/utils.py

```diff
@@ -55,38 +78,117 @@ def update_release_from_github(release, repo_api=None):
     return release
 
 
-def send_release_webhook(project_config, release):
+def send_release_webhook(project_config, release, config_item=None):
     if release is None or not settings.METACI_RELEASE_WEBHOOK_URL:
-        return
+        return  # should we better error handle this?
     logger.info(
         f"Sending release webhook for {release} to {settings.METACI_RELEASE_WEBHOOK_URL}"
     )
     tag = release.git_tag
+
+    steps = []
+    if config_item and settings.METACI_START_STOP_WEBHOOK:
+        implementation_steps = release.implementation_steps.all()
+        steps = [
+            implementation_payload(implementation_step.plan.role, config_item)
+            for implementation_step in implementation_steps
+        ]
+
     payload = {
         "case_template_id": release.change_case_template.case_template_id,
         "package_name": project_config.project__package__name,
         "version": project_config.get_version_for_tag(tag),
         "release_url": f"{release.repo.url}/releases/tag/{urllib.parse.quote(tag)}",
+        "steps": steps,
     }
-    token = jwt.encode(
-        {
-            "iss": settings.METACI_RELEASE_WEBHOOK_ISSUER,
-            "exp": timegm(datetime.utcnow().utctimetuple()),
-        },
-        settings.METACI_RELEASE_WEBHOOK_AUTH_KEY,
-        algorithm="HS256",
-    )
+    token = jwt_for_webhook()
     response = requests.post(
-        settings.METACI_RELEASE_WEBHOOK_URL,
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/release/",
         json=payload,
         headers={"Authorization": f"Bearer {token}"},
     )
```

Comment:
Note to self: I need to be sure to adjust METACI_RELEASE_WEBHOOK_URL to remove /release/ when I deploy this.